src/chrome/content/rules/Asciinema.org.xml

```diff
@@ -46,6 +36,9 @@ Fetch error: http://www.asciinema.org/ => https://www.asciinema.org/: (51, "SSL:
 	<rule from="^http://blog\.asciinema\.org/"
```

Comment:
Change `<securecookie host="^asciinema\.org$" name=".+" />` to `<securecookie host=".+" name=".+" />` unless securing only the top domain cookies is your intent.